packages/sdk/__tests__/__unit__/utils/Utils.unit.test.ts

```diff
@@ -398,12 +398,12 @@ describe('Utils - enforceParentheses', () => {
 
   it("should add first bracket when end exists", () => {
     const output = Utils.enforceParentheses("input with spaces)");
-    expect(output).toEqual("(input with spaces)");
+    expect(output).toEqual("(input with spaces))");
```

Comment:
I haven't tested this yet, but it doesn't seem like having an extra closing `)` will work in some scenarios 🤔